cms/src/components/content/ContentOverview.vue

```diff
@@ -77,12 +78,87 @@ if (!Object.entries(TagType).some((t) => t[1] == tagTypeString)) tagTypeString =
 
 const titleType = tagTypeString ? tagTypeString : props.docType;
 router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} overview`;
+
+const showFilterOptions = ref(false);
+
+const filterByTranslation = ref(queryOptions.value.translationStatus);
+const filterByTranslationOptions = [
+    {
+        value: "translated",
+        label: "Translated",
+    },
+    {
+        value: "untranslated",
+        label: "Untranslated",
+    },
+    {
+        value: "all",
+        label: "All",
+    },
+];
+
+const filterByStatus = ref(queryOptions.value.publishStatus);
+const filterByStatusOptions = [
+    {
+        value: "published",
+        label: "Published",
+    },
+    {
+        value: "scheduled",
+        label: "Scheduled",
+    },
+    {
+        value: "expired",
+        label: "Expired",
+    },
+    {
+        value: "draft",
+        label: "Drafted",
+    },
+    {
+        value: "all",
+        label: "All",
+    },
+];
+
+watch(filterByTranslation, () => {
+    queryOptions.value.translationStatus = filterByTranslation.value as
+        | "all"
+        | "translated"
+        | "untranslated"
+        | undefined;
+});
+
+watch(filterByStatus, () => {
+    queryOptions.value.publishStatus = filterByStatus.value as
+        | "all"
+        | "published"
+        | "scheduled"
+        | "expired"
+        | "draft"
+        | undefined;
+});
+
+const handleShowFilterOptions = () => {
+    showFilterOptions.value = !showFilterOptions.value;
+};
 </script>
 
 <template>
     <BasePage :title="`${capitaliseFirstLetter(titleType)} overview`">
         <template #actions>
             <div class="flex gap-4">
+                <button
+                    @click="handleShowFilterOptions"
+                    class="rounded-md border-[1px] p-2 px-3 shadow-sm"
+                    :style="{
+                        backgroundColor: showFilterOptions ? '#18181b' : 'white',
+                        color: showFilterOptions ? 'white' : '#18181b',
```

Comment:
Yeah maybe you could help me with this. As this was the only method I found that worked for this part of the code. Could be nice to learn what a better way could be

Reply:
I actually think when we merge it, inverting the colors will look inconsistent with the sort button, so that part of the visual should be added when merged if we decide on it.